.travis.yml

```diff
@@ -50,7 +50,7 @@ before_install:
   - "travis_retry pip install twine wheel coveralls requirements-builder"
   - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
   - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder --level=dev --req requirements-devel-cernopendata.txt setup.py > .travis-devel-requirements.txt"
```

Comment:
Yes, and I was also going to separate `docker-compose.yml` from `docker-compose-dev.yml`later...